src/utils/latency_tracer.cpp

```diff
@@ -161,6 +162,20 @@ void latency_tracer::append_point(const std::string &stage_name, uint64_t timest
     uint64_t cur_ts = timestamp > _last_time ? timestamp : _last_time + 1;
     _points.emplace(cur_ts, stage_name);
     _last_time = cur_ts;
+    _last_stage = stage_name;
+}
+
+void latency_tracer::add_sub_tracer(const std::string &name)
+{
+    if (!_enable_trace) {
+        return;
+    }
+
+    auto sub_tracer = std::make_shared<dsn::utils::latency_tracer>(true, name, 0);
```

Comment:
should we use unique_ptr for `sub_tracer`?

Reply:
sub_tracer may be shared, in this pr, the `slog` tracer is shared by different mutation